backends/cuda-reg/ceed-cuda-reg.c

```diff
@@ -20,6 +20,11 @@
 #include "ceed-cuda-reg.h"
 #include "../cuda/ceed-cuda.h"
 
+static int CeedGetPreferredMemType_Cuda_reg(CeedMemType *type) {
```

Comment:
I fixed the delegation, so this shouldn't be needed anymore.

Reply:
done.